web/src/components/Testimonial/Testimonial.module.scss

```diff
@@ -42,8 +42,8 @@
     }
 
     @container testimonial (min-width: #{media-queries.$lg}) {
-      padding-left: utils.rem(spacing.$xx-large); // 1rem L/R margin from layout.container
-      padding-right: utils.rem(spacing.$xx-large); // 1rem L/R margin from layout.container
+      padding-left: utils.rem(spacing.$large); // 1rem L/R margin from layout.container
+      padding-right: utils.rem(spacing.$large); // 1rem L/R margin from layout.container
```

Comment:
This works out as 3rem because the 1rem L/R margin gets added to the padding; if we're wanting 2rem we don't need to set these as its the same as `sm` breakpoint (L40-41)